doc/developer_guide/index.md

```diff
@@ -85,24 +105,38 @@ Syncing the git tags can be done with:
 pixi run sync-git-tags
 ```
 
-### Editable install
+## Developer Environment
 
-It can be advantageous to install the HoloViews in [editable mode](https://pip.pypa.io/en/stable/topics/local-project-installs/#editable-installs):
+The `default` environment is meant to provide all the tools needed to develop HoloViews.
 
-```bash
-pixi run install
-```
+This environment can be created by running `pixi run install`, which will set up the environment and make an editable install of HoloViews.
```

Comment:
The docs already mention `pixi run install`, that feels a little redundant. I'd just remove it.